src/components/Comment/CommentList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { CommentInfo } from './CommentInfo';
+import { Comment } from '../User';
+
+type Props = {
+  comment: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comment }) => (
+  <ul className="CommentList" data-cy="comments-list">
+    {comment.map(comm => (
+      <li className="PostList__items">
```

Comment:
Where is your key value for list? Console will show you an error becouse of unique keys
[https://uk.reactjs.org/docs/lists-and-keys.html](url)